src/main/java/com/algorand/algosdk/v2/client/model/ApplicationParams.java

```diff
@@ -22,7 +22,6 @@
     public void approvalProgram(String base64Encoded) {
         this.approvalProgram = Encoder.decodeFromBase64(base64Encoded);
     }
-    @JsonProperty("approval-program")
```

Comment:
I'll open one for generator, removing this annotation allows us to pass the bytes for the approval/clear program as bytes when we msgpack encode them rather than getting the b64 encoded version.

Reply:
Let me take a look soon...

Reply:
Ping me once this is opened, and hopefully nothing's broken 😅